docs/reference/migration/migrate_7_0/logging.asciidoc

```diff
@@ -31,3 +31,10 @@ Note: GC logs which are written to the file `gc.log` will not be changed.
 ==== Docker output in JSON format
 
 All Docker console logs are now in JSON format. You can distinguish logs streams with the `type` field.
+
+[float]
+==== Audit plaintext log file removed, JSON file renamed
+
+Elasticsearch will no longer produce a plaintext audit log file `${cluster_name}_access.log`.
```

Comment:
The "will" makes me wonder if this is something happening in the future or is already true for this version. If the latter, I'd change it to "Elasticsearch no longer produces...".